src/layouts/shared/footer.tsx

```diff
@@ -0,0 +1,282 @@
+            </h4>
+
+            <ul className={style.footer__linkList}>
+              {helpPageUrl ? (
+                <li>
+                  <a
+                    href={
+                      language === Language.English && helpPageUrlEnglish
+                        ? helpPageUrlEnglish
+                        : helpPageUrl
+                    }
+                    target="_blank"
+                    rel="noreferrer"
+                  >
+                    {t(
+                      ModuleText.Layout.base.footer.sections.general
+                        .helpPageLink,
+                    )}
+                  </a>
+                </li>
+              ) : null}
+            </ul>
+          </section>
+
+          <section className={style.footer__section}>
+            <h4 className={style.footer__title}>
+              {t(ModuleText.Layout.base.footer.sections.contact.header)}
+            </h4>
+
+            <ul className={style.footer__linkList}>
+              <li>
+                <Link href="/contact">
+                  {t(
+                    ModuleText.Layout.base.footer.sections.contact.contactLink,
+                  )}
+                </Link>
+              </li>
+            </ul>
+          </section>
+
+          {!withoutSettings && (
+            <section className={style.footer__section}>
+              <h4 className={style.footer__title}>
+                {t(ModuleText.Layout.base.footer.sections.settings.header)}
+              </h4>
+
+              <ul className={style.footer__linkList}>
+                <LanguageSelections />
+                <li>
+                  <button
+                    className={style.buttonLink}
+                    onClick={() => toggleDarkmode(!isDarkMode)}
+                  >
+                    {isDarkMode
+                      ? t(
+                          ModuleText.Layout.base.footer.sections.settings
+                            .toggleLightMode,
+                        )
+                      : t(
+                          ModuleText.Layout.base.footer.sections.settings
+                            .toggleDarkMode,
+                        )}
+                  </button>
+                </li>
+              </ul>
+
+              <p className={style.footer__cookieWarning}>
+                {t(
+                  ModuleText.Layout.base.footer.sections.settings
+                    .cookiesWarning,
+                )}
+              </p>
+            </section>
+          )}
+        </div>
+
+        <hr />
+
+        <section className={style.footer__bottomLinks}>
+          <div className={style.footer__some}>
+            {facebookLink && (
+              <ButtonLink
+                mode={isDarkMode ? 'interactive_2' : 'interactive_1'}
+                radius="top-bottom"
+                display="inline"
+                radiusSize="circular"
+                icon={{ left: <img src="/fb.svg" alt="Facebook" /> }}
```

Comment:
Looks like these social media icons are missing

Reply:
True! Do we want them so I should add them now? Probably?

Reply:
They are at least in the Figma sketches, so my guess is that we should have them. If not, we should probably remove the links altogether 🤔

Reply:
I can add them 👍

Reply:
It seems like it's copy paste from the webshop though (we are upgrading link etc.), so not sure how thought-through it is 🤔

Reply:
Faster to remove at a later point than add them now that it is being worked on.

Reply:
True 👍